pom.client.xml

```diff
@@ -1263,6 +1263,7 @@
     <module>sdk/core/azure-core-http-netty</module>
     <module>sdk/core/azure-core-http-okhttp</module>
     <module>sdk/core/azure-core-test</module>
+    <module>common/perf-test-core</module>
```

Comment:
Please put this alphabetically in the module list. There's no reason to break up the core modules with this entry.

Reply:
done